app/src/main/java/io/homeassistant/companion/android/webview/WebViewPresenterImpl.kt

```diff
@@ -70,6 +73,15 @@ class WebViewPresenterImpl @Inject constructor(
         }
     }
 
+    override fun isFS(key: String): Boolean {
```

Comment:
Not sure why this is taking a parameter?  Can we just rename to `isFullScreen()`?

Reply:
isFS () function parameter has been removed